scripts/apps/authoring/views/item-association.html

```diff
@@ -7,10 +7,15 @@
         </button>
     </div>
     <video ng-if="related && related.type === 'video'" controls="controls">
-        <source vsrc="{{ rendition.href }}"
+        <source src="{{ rendition.href }}"
                 ng-repeat="(key, rendition) in related.renditions"
                 ng-if="isVideo(rendition)" html5vfix>
     </video>
+    <audio ng-if="related && related.type === 'audio'" controls="controls">
+        <source src="{{ rendition.href }}"
```

Comment:
I think it would be better to use [ngSrc](https://docs.angularjs.org/api/ng/directive/ngSrc)